examples/src/main/scala/org/apache/spark/examples/graphx/Analytics.scala

```diff
@@ -51,7 +51,7 @@ object Analytics {
         case _ => throw new IllegalArgumentException(s"Invalid argument: $arg")
       }
     }
-    val options = mutable.Map(optionsList: _*)
+    val options = mutable.Map(immutable.ArraySeq.unsafeWrapArray(optionsList): _*)
```

Comment:
sorry, https://github.com/apache/spark/pull/43642/files#r1383391358 should for this case

Reply:
I'm confused, why don't we use `org.apache.spark.util.ArrayImplicits` here?

Reply:
`ArrayImplicits` is currently in the `private[spark]` scope, should we expose them in the examples code? Sorry, I'm not very clear about this rule.

Reply:
> oh, because `examples` do not depend on common/utils

`common/utils` is a transitive dependency of the `core` module, so `ArrayImplicits ` is visible to the `examples` module, I previously suggested this only because I believe that `private[spark]` code should not be part of the examples. In this context, what is your suggestion? @cloud-fan

Reply:
> oh, because `examples` do not depend on common/utils

Just to clarify, the `mllib-local` module indeed does not depend on the `common/utils` module, so `ArrayImplicits ` is not used in the `mllib-local` module.

Reply:
It's fine to make `examples` an exception.

Reply:
Thanks for your confirmation.